rep-0003.rst

```diff
@@ -220,18 +216,64 @@ Exact or Series Requirements:
 - Qt 5.3.x
 - PyQt5
 
-Buildsystem support:
+Build System Support:
 
-- catkin:
+- Same as Indigo
 
-  - build from source
-  - release for binary packaging
-  - wiki documentation
-  - continuous integration
+Lunar Loggerhead (May 2017 - May 2019)
+--------------------------------------
+Required Support for:
 
-- rosbuild:
+- Ubuntu Xenial (16.04)
+- Ubuntu Yakkety (16.10)
+- Ubuntu Zesty (17.04)
 
-  - build from source
+Recommended Support for:
+
+- Debian Stretch
+- Fedora 26
+
+Architectures Supported:
+
+- amd64
+- arm32
+- arm64
+
+Minimum Requirements:
+
+ Most of the following requirements are the same as Kinetic's to avoid maintainers
+ branching out before the next LTS release
+
+- C++11
+- Python 2.7
+
+  - Python 3.5 not required, but testing against it is recommended
+
+- Lisp SBCL 1.2.4
+- CMake 3.0.2
+- Boost 1.55
+
+Exact or Series Requirements:
+
+- Ogre3D 1.9.x
+- Gazebo 8 (packaged by OSRF)
+
+  - ROS won't use upstream Gazebo for the upcoming few releases because of a mismatch
+    between upstream versions on targeted Ubuntu distributions
+  - In the case of Lunar:
+
+    - Zesty ships with Gazebo 8
+    - Yakkety and Xenial ship with Gazebo 7
+
+- PCL 1.7.x (Xenial)
+- PCL 1.8.x (Other Systems)
```

Comment:
> Lunar is short term support distribution and developers are expecting to use kinetic-branch as much as possible, which was targeting to PCL 1.7.

Right, the main goal is to allow people to use the kinetic-branch until the next ROS LTS (ROS-M).

> All PCL 1.7 code can compile and run against PCL 1.8 (Am I correct?)

Yes, from the PCL [changelog](https://github.com/PointCloudLibrary/pcl/blob/757e28a75a0c6c0413085a2af070670828a48527/CHANGES.md) it seems to be mostly bug fixing and feature addition.

> Although some of Lunar target platform uses PCL 1.8, developer should build their Lunar package against PCL 1.7.

Developpers should build their packages against the system version of PCL. They can use PCL1.8 features as long as there is a code path to build against 1.7 (as it's currently done for leveraging c++11 features for example)

> If you want to use PCL 1.8, wait for next ROS-M release

Right, ROS-M will target Ubuntu 18.04 and the latest stable Debian released at that time. It will target the lowest version of PCL that is released on these architecture, very likely 1.8.